src/test/java/org/junitpioneer/jupiter/resource/MorePaths.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2016-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junitpioneer.jupiter.resource;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+final class MorePaths {
+
+	static Path rootTempDir() {
```

Comment:
It's a little dirty, but I think we can move this into `Assertions.java` (in this package) to get rid of this class. And you could make it a public static final field, initiated to the system property.

Reply:
That sounds sensible! Done: https://github.com/junit-pioneer/junit-pioneer/pull/491/commits/eeb123eb2367b0b29d4b596d40f6f73b537a818e